### PR TITLE
add iframe extension

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

**Comment by benbarnett** [comment]
What's the iframe for?

**Comment by dougdroper** [comment]
:+1:

File: lib/mastalk/extensions.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ def extensions
 
       Dir.foreach(SNIPPETS_FOLDER) do |file|
         next if file == '.' || file == '..'
+        next unless file.end_with?('.erb')
```
Comment (benlovell) [inline lib/mastalk/extensions.rb]: I'd favour `Dir.glob` with a wildcard match for `*.erb` over `Dir.foreach` and this logic.

**Comment by asmega** [comment]
@benbarnett in the very near future it will iframe https://www.moneyadviceservice.org.uk/en/tools/divorce-and-separation-calculator

**Comment by benbarnett** [comment]
That's Flash...

**Comment by asmega** [comment]
@benbarnett correct

**Comment by benbarnett** [comment]
...

**Comment by asmega** [comment]
Better or worse for the Dir.glob change?

It's like being at the opticians: first or second; red or green.

**Comment by benlovell** [comment]
:eyeglasses: I'd say it was marginally better, and more efficient so :ship: :smile:

File: lib/mastalk/extensions.rb [hunk 2]
```diff
@@ -22,9 +23,9 @@ def extension(start, stop = nil, &block)
     def extensions
       return @@extensions unless @@extensions.empty?
 
-      Dir.foreach(SNIPPETS_FOLDER) do |file|
+      Dir.glob(SNIPPET_FILES).each do |file|
         next if file == '.' || file == '..'
```
Comment (dougdroper) [inline lib/mastalk/extensions.rb]: Can you remove this line now that you are using Dir.glob?

Comment (asmega) [inline lib/mastalk/extensions.rb]: yep. done

**Comment by ourmaninamsterdam** [comment]
:+1:

**Comment by asmega** [comment]
turns out this feature is no longer required. so closing. someone can re-open it if they need iframes

**Comment by benbarnett** [comment]
*the crowd cheers*

**Pull request closed without merging**